arbor/backends/gpu/matrix_fine.cu

```diff
@@ -252,7 +258,7 @@ void solve_matrix_fine(
 
 void gather(
     const fvm_value_type* from,
-    fvm_value_type* to,
+    fvm_value_type*  to,
```

Comment:
can you undo the white space changes? they occur in a few more spots.

Reply:
👍